inf-haskell.el

```diff
@@ -81,7 +81,7 @@ The command can include arguments."
   ;; The format of error messages used by Hugs.
   `(("^ERROR \"\\(.+?\\)\"\\(:\\| line \\)\\([0-9]+\\) - " 1 3)
     ;; Format of error messages used by GHCi.
-    ("^\\(.+?\\):\\([0-9]+\\):\\(\\([0-9]+\\):\\)?\\( \\|\n *\\)\\(Warning\\)?"
+    ("^\\(.+?\\):\\([0-9]+\\):\\(\\([0-9]+\\):\\)?\\( \\|\n *\\)\\(.arning\\)?"
```

Comment:
That was my first idea, but there is so much escaping going on in the regex I was not sure if any would be needed.  Will update.